app/assets/stylesheets/application.scss

```diff
@@ -25,7 +25,7 @@
   flex: none;
   flex-wrap: wrap;
   .card {
-    width: 300px;
+    width: 245px;
```

Comment:
I don't think we should be setting the widths of `card` here - we should really be using bootstrap widths